docs/documentation/admin/workflows/admin_workflow_groups.md

```diff
@@ -0,0 +1,264 @@
+---
+id: admin_workflow_groups
+title: I. Workflow Groups
```

Comment:
@Mauriceac does this work with i18n?

Reply:
Yes, we just have to copy all the files into the i18n folder and maintain the IDs.